modules/apps/foundation/frontend-image-editor/frontend-image-editor-web/.task-cache/resources/ImageEditor.scss

```diff
@@ -0,0 +1,220 @@
+@import "bourbon";
+@import "mixins";
+
+$imageeditor-btn-padding: 8px 4px !default;
+$imageeditor-btn-space: 24px !default;
+$imageeditor-controls-height: 52px !default;
+$imageeditor-filters-border-width: 4px;
+$imageeditor-filters-height: 90px !default;
+$imageeditor-history-controls-top: 30px;
+$slider-height: 20px;
+
+// Skin
```

Comment:
We can probably drop the comments, since all of the CSS below is a mix of skin and structure. Or does this only apply to the variables?
If so, can we change this comment to `Skin Variables` so it's clear what it applies to?